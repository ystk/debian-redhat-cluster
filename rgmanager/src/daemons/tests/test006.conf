<?xml version="1.0"?>
<!-- The IP address should have a NEEDSTOP in the old tree and NEEDSTART
     in the new tree, -->
<cluster>
<rm>
	<resources>
    		<service name="test1"/>
		<script name="initscript" file="/etc/init.d/sshd"/>
		<ip address="192.168.1.2" monitor_link="yes"/>
	</resources>
	<service ref="test1">
		<ip ref="192.168.1.2"/>
		<script ref="initscript"/>
	</service>
</rm>
</cluster>
