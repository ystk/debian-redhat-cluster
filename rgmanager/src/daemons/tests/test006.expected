=== Resources List ===
Resource type: ip
Instances: 1/1
Agent: ip.sh
Attributes:
  address = 192.168.1.2 [ primary unique ]
  monitor_link = yes
  nfslock [ inherit("service%nfslock") ]

Resource type: script
Agent: script.sh
Attributes:
  name = initscript [ primary unique ]
  file = /etc/init.d/sshd [ unique required ]
  service_name [ inherit("service%name") ]

Resource type: service
Instances: 1/1
Agent: service.sh
Attributes:
  name = test1 [ primary unique required ]

=== Resource Tree ===
service {
  name = "test1";
  ip {
    address = "192.168.1.2";
    monitor_link = "yes";
      }
  script {
    name = "initscript";
    file = "/etc/init.d/sshd";
    service_name = "test1";
  }
}
